backend/tests/integration/test_ttl_cache_behavior.py

```
@@ -0,0 +1,249 @@
+        """Test cache behavior when approaching maxsize limits."""
+        with patch('tarsy.services.alert_service.TTLCache') as mock_ttl_cache:
+            # Create caches with very small maxsize for testing
+            mock_ttl_cache.side_effect = lambda maxsize, ttl: TTLCache(maxsize=3, ttl=3600)
+            
+            service = AlertService(mock_settings)
+            
+            # Fill caches beyond capacity
+            alert_ids = [f"alert-{i}" for i in range(5)]
+            session_ids = [f"session-{i}" for i in range(5)]
+            
+            for alert_id, session_id in zip(alert_ids, session_ids):
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)